lambdas/process_webhooks/process_webhooks.py

```diff
@@ -21,20 +21,36 @@
 logger.setLevel(numeric_level)
 
 
-def _get_target_url():
+def _get_target_url(headers):
     """
-    Get the target URL for the processed hooks from a comma delimited
-    list of URL, set in an OS environment variable.
+    Get the target URL for the processed hooks from the
+    OS environment variable. Based on the GitHub event,
+    add the proper endpoint.
 
-    Return the target URL
+    Return the target URL with the appropriate endpoint
     """
     url = os.environ.get('TARGET_URL')
     if not url:
         raise StandardError(
             "Environment variable TARGET_URL was not set"
         )
 
-    return url
+    event_type = headers.get('X-GitHub-Event')
+
+    # Based on the X-Github-Evenet header, determine the
+    # proper endpoint for the target url.
+    # PR's and Issue Comments use the ghprb Jenkins plugin
+    # Pushes use the Github Jenkins plugin
+    if event_type in ["issue_comment", "pull_request"]:
+        endpoint = "ghprbhook/"
+    elif event_type == "push":
+        endpoint = "github-webhook/"
+    else:
+        raise StandardError(
```

Comment:
Question: should we be throwing an exception here? It will trigger cloudwatch alerts. Maybe this makes sense because we should never have other event types being sent to the spigot. What do you think?